### PR TITLE
Capitalise "Sign out"

### PR DESCRIPTION
**Pull request by PoIygon**
### Changes

I have changed the "Capitalize Account Settings" add-on so it also capitalizes "Sign out" button.
### Reason for changes

The changes should be made because the "Sign out" button is also not capitalized and it doesn't fit with the other buttons.
### Tests

I have tested this pull request by using the "load temporary add-on" feature in Firefox.

### CODE REVIEW

**Comment by WorldLanguages** [comment]
@mxmou What do you think? Is there a specific reason you didn't do this too when submitting this addon initially?

File: addons/account-settings-capitalize/addon.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Capitalized Account Settings",
-  "description": "Changes \"Account settings\" in the account menu to \"Account Settings\".",
+  "name": "Capitalized Account Settings and Sign Out",
```
Comment (RedGuy12) [inline addons/account-settings-capitalize/addon.json:2]:
```suggestion
  "name": "Capitalized account menu items",
```

File: addons/account-settings-capitalize/addon.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "description": "Changes \"Account settings\" and \"Sign out\" in the account menu to \"Account Settings\" and \"Sign Out\".",
```
Comment (RedGuy12) [inline addons/account-settings-capitalize/addon.json:3]:
```suggestion
  "description": "Changes \"Account settings\" and \"Sign out\" in the account menu to \"Account Settings\" and \"Sign Out\", respectively.",
```

File: addons/account-settings-capitalize/style.css [hunk 3]
```diff
@@ -1,4 +1,5 @@
 .account-nav .dropdown > li:nth-child(3),
-ul.user-nav > li:nth-child(3) {
+ul.user-nav > li:nth-child(3), .account-nav .dropdown > li:nth-child(3),
```
Comment (RedGuy12) [inline addons/account-settings-capitalize/style.css:2]: i think this is supposed to be
```suggestion
ul.user-nav > li:nth-child(3), .account-nav .dropdown > li:nth-child(4),
```
?

**Review by RedGuy12 — CHANGES_REQUESTED** [review]
maybe this addon could also capitalize `Sign in` in the navbar?

**Comment by mxmou** [comment]
@WorldLanguages "Sign in" and, in the editor, "Fade in" and "Fade out" aren't capitalized either, so this would just make things less consistent.

**Comment by PoIygon** [comment]
This is not active at all

**Comment by WorldLanguages** [comment]
I'm unsure whether this change makes it more consistent or not.
`Account Settings` is obvious because the page title is title case:
image
But if we capitalized `Sign Out`, then why not also capitalize `Sign in`? And we have an endless chain of consistency.

**Pull request closed without merging**